Dockerfile

```diff
@@ -1,23 +1,19 @@
-FROM debian:9
+FROM python:3.5-stretch
 LABEL maintainer="Codimp"
 
-RUN apt-get update && \
-    apt-get install -y \
-    dexdump \
-    python3-dev \
-    python3-pip \
-    libssl-dev \
-    libffi-dev \
-    libxml2-dev \
-    libxslt1-dev \
-    postgresql-client
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y \
+    dexdump=7.0.0* \
+    postgresql-client-9.6=9.6.15* \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
Purely aesthetic question, why not:
```
RUN apt-get update && \
    apt-get install --no-install-recommends -y \
    dexdump=7.0.0* \
    postgresql-client-9.6=9.6.15* && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```
or even:
```
RUN apt-get update && \
    apt-get install --no-install-recommends -y dexdump=7.0.0* postgresql-client-9.6=9.6.15* && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

(I like the second option better)

Reply:
Here you go!